src/tests/python/sssd/testlib/common/utils.py

```diff
@@ -728,7 +742,7 @@ def createselfsignedcerts(self,
                               serverlist,
                               ca_dn=None,
                               passphrase='Secret123',
-                              canickname='Example CA'):
+                              canickname='ExampleCA'):
```

Comment:
This commit does two different things. It adds a new function and then fixes the CA name. Each change should be in a separate commit.